http/openapi.go

```diff
@@ -111,7 +111,8 @@ func NewOpenAPISpec() (*openapi3.T, error) {
 		Servers: openapi3.Servers{
 			&openapi3.Server{
 				Description: "Local DefraDB instance",
-				URL:         "http://localhost:9181/api/v0",
+				// This will be overriden by the ServerAddressMiddleware
+				URL: "http://127.0.0.1:9181/api/v0",
```

Comment:
suggestion: As stated above, we can change this to `URL: "/api/v0"`. The client will treat it as a relative path to the playground and it will work just fine.

Reply:
I just tested and confirmed that this works, so I think it is also the cleanest way to do it.